howso/utilities/internals.py

```diff
@@ -566,8 +568,8 @@ def sanitize_for_json(obj: Any):  # noqa: C901
     elif hasattr(obj, 'openapi_types') and hasattr(obj, 'attribute_map'):
         # Convert openapi model to dict using the attribute mapping,
         # excluding values which are None
-        obj_dict = {obj.attribute_map[attr]: getattr(obj, attr)
-                    for attr, _ in obj.openapi_types.items()
+        obj_dict = {ATTRIBUTE_MAP[attr]: getattr(obj, attr)
```

Comment:
This should not use the globally defined ATTRIBUTE_MAP but the `obj.attribute_map`

Reply:
Addressed by: 2ba04a5